modules/flowable-cmmn-converter/src/main/java/org/flowable/cmmn/converter/StandardEventXmlConverter.java

```diff
@@ -34,12 +36,16 @@ public boolean hasChildElements() {
 
     @Override
     protected BaseElement convert(XMLStreamReader xtr, ConversionHelper conversionHelper) {
-        String event = xtr.getText();
-        if (conversionHelper.getCurrentCmmnElement() instanceof TimerEventListener) {
-            TimerEventListener timerEventListener = (TimerEventListener) conversionHelper.getCurrentCmmnElement();
-            timerEventListener.setTimerStartTriggerStandardEvent(event);
-        } else {
-            conversionHelper.getCurrentSentryOnPart().setStandardEvent(xtr.getText());
+        try {
+            String event = xtr.getElementText();
+            if (conversionHelper.getCurrentCmmnElement() instanceof TimerEventListener) {
+                TimerEventListener timerEventListener = (TimerEventListener) conversionHelper.getCurrentCmmnElement();
+                timerEventListener.setTimerStartTriggerStandardEvent(event);
+            } else {
+                conversionHelper.getCurrentSentryOnPart().setStandardEvent(xtr.getText());
```

Comment:
I think that here we need to use the `event` that we got from the `xtr.getElementText()` call on line 40